pkg/agent/datamodel/const.go

```diff
@@ -98,3 +98,9 @@ const (
 	// AADPodIdentityAddonName is the name of the aad-pod-identity addon deployment
 	AADPodIdentityAddonName = "aad-pod-identity"
 )
+
+const (
+	// TempDiskContainerDataDir is the path used to mount docker images, emptyDir volumes, and kubelet data
+	// when KubeletDiskType == TempDisk.
+	TempDiskContainerDataDir = "/mnt/aks/containers"
```

Comment:
basically just need something under /mnt that is unique to aks. we will eventually put other things here, so I'm reserving the `/mnt/aks` prefix for all our content, and specifically taking `/mnt/aks/containers` for the container data dir.

Reply:
If this is used for temp disk, would put temp disk somewhere in the name be helpful in the long run? /mnt/aks/containers sounds like something very official and important. If you want to reserve /mnt/aks for various use, how about /mnt/aks/kubelettempdisk?

Reply:
/mnt is where Azure puts the temp disk by default, we are merely claiming a "slice" of /mnt for aks :)

Reply:
the subdirectories would be /mnt/aks/kubelet and /mnt/aks/containers for data